packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1141,11 +1155,29 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
     return child;
   }
 
+  /// Whether this route should ignore pointers when it is the current route and
+  /// transitions are in progress.
+  ///
+  /// Pointers always are ignored when [isCurrent] is false (e.g., when a route
+  /// has a new route pushed on top of it, or during a route's exit transition
+  /// after popping). Override this value to also ignore pointers on pages during
+  /// transitions where this route is the current route (e.g., after the route
+  /// above this route pops, or during this route's entrance transition).
+  ///
+  /// This value should generally be true for Cupertino routes.
+  ///
+  /// Returns false by default.
+  ///
+  @protected
+  bool get ignorePointerWhenCurrentDuringTransitions => false;
+
   @override
   void install() {
     super.install();
-    _animationProxy = ProxyAnimation(super.animation);
-    _secondaryAnimationProxy = ProxyAnimation(super.secondaryAnimation);
+    _animationProxy = ProxyAnimation(super.animation)
+      ..addStatusListener((_) => _maybeUpdateIgnorePointer());
+    _secondaryAnimationProxy = ProxyAnimation(super.secondaryAnimation)
+      ..addStatusListener((_) => _maybeUpdateIgnorePointer());
```

Comment:
Statuses aren't passed into the function here because we need to consult the status for both animations when deciding whether to ignore pointers.

Also, I considered naming this something like `_handleStatusChanged` or `_handleAnimationStatusChanged`, but `_handleStatusChanged` is already defined in `TransitionRoute`

Reply:
our style guide suggests `Avoid anonymous parameter names`

Reply:
I would probably create an private method instead of arrow function due to the length

Reply:
Gotcha. I added `AnimationStatus status` as a parameter to `_maybeUpdateIgnorePointer` which is a private function so this can just be a tear-off—is that what you meant?